addons/jam/CfgMagazineWells.hpp

```diff
@@ -32,7 +32,7 @@ class CfgMagazineWells {
 
     // Pistol calibre magwells, ordered lexicographically in metric and imperial groups
     #include "magwells_10mmAuto.hpp"    // 10mm Auto | 10mm Automatic | 10x25mm
-    #include "magwells_570x28.hpp"      // 5.70x28mm
+    #include "magwells_57x28.hpp"       // 5.7x28mm
```

Comment:
This one is from 3.9 or earlier, right?
```suggestion
    #include "magwells_570x28.hpp"      // 5.7x28mm
```
Have to keep it for bwc.

Reply:
No, it's from #1061.

Reply:
Then I support changing this now.